### PR TITLE
CodeCamp #156

### PR DESCRIPTION
**Pull request by cocoshe**

### CODE REVIEW

**Comment by ly015** [comment]
Thank you very much for helping refine MMAction2's documentation.

**Review by Dai-Wenxun — COMMENTED** [review]
Since we updated get_started.md in #2058 recently, could you please refine your translation according to our modifications? The latest get_started.md can be found [here](https://github.com/open-mmlab/mmaction2/blob/dev-1.x/docs/en/get_started.md). Thanks!

**Review by Dai-Wenxun — APPROVED** [review]

File: docs/zh_cn/get_started.md [hunk 1]
```diff
@@ -1,3 +1,186 @@
...
+print('The top-5 labels with corresponding scores are:')
+for result in results:
+    print(f'{result[0]}: ', result[1])
+```
+
+## 自定义安装
+
+### CUDA 版本
+
+安装PyTorch时，你可能需要提供特定的CUDA的版本。如果你不清楚选择哪个版本，请遵循我们的建议：
```
Comment (ly015) [inline docs/zh_cn/get_started.md:140]:
```suggestion
安装 PyTorch 时，你可能需要安装特定的 CUDA 的版本。如果你不清楚应该选择哪个版本，请遵循我们的建议：
```

File: docs/zh_cn/user_guides/3_inference.md [hunk 2]
```diff
@@ -1 +1,41 @@
-# 教程 3：使用现有模型推理（内容建设中）
+# 教程3：利用现有模型进行推理
+
+MMAction2在[Model Zoo](../modelzoo.md)中提供预训练的视频理解模型。
```
Comment (ly015) [inline docs/zh_cn/user_guides/3_inference.md:3]:
```suggestion
MMAction2 在 [Model Zoo](../modelzoo.md) 中提供预训练的视频理解模型。
```

**Pull request merged**